docs/mkdocs.yml

```diff
@@ -19,18 +19,18 @@ theme:
     custom_dir: overrides
 
 google_analytics:
-  - UA-104326577-3
-  - auto
+    - UA-104326577-3
+    - auto
 
 plugins:
     - search
     - awesome-pages
 nav:
     - index.md
     - Cookbooks:
-        - Common workflows: cookbooks/common_workflows.md
-        - Creating a derived dataset: cookbooks/creating_a_derived_dataset.md
+          - Common workflows: cookbooks/common_workflows.md
+          - Creating a derived dataset: cookbooks/creating_a_derived_dataset.md
```

Comment:
Why this much indentation here? I'd prefer to see this whole file use 2 spaces, which would be more consistent with other files in this repo.

Reply:
When filing this PR I went with whatever involved the smallest change. For the linting rules that meant consistent indent, rather than strict indent. For this file that meant 4-space indent. I'm happy to change this file to 2-space indent.